src/Util/YamlSourceManipulator.php

```diff
@@ -461,12 +461,30 @@ private function changeValueInYaml($value)
             // empty space between key & value
             $newYamlValue = ' '.$newYamlValue;
         }
+
+        $newPosition = $this->currentPosition + \strlen($newYamlValue);
+        $isNextContentComment = $this->isPreviousLineComment($newPosition);
+        if ($isNextContentComment) {
+            $newPosition++;
+        }
```

Comment:
Shouldn't it be a `while` in case of multi-line comment?

Reply:
The `$newPosition++` here is just because of the code below that adds *1* new character (`\n`) if the next line is a comment. I'm moving this pointer/position forward 1 for that 1 `\n`. It's confusing code :). The test case actually does have 2 lines of comment.